configuration/pipeline_config.json

```diff
@@ -40,12 +40,26 @@
         "3903ad8e-284e-48a0-881b-89c0b06b9334",
         "45939459-e4d5-4ad6-8650-3e7134b9a035"
       ]
+    },
+    {
+      "SourceType": "RecoveryCSV",
+      "ActivationFlowURLs": [
+        "gs://avf-project-datasets/2020/IMAQAL-COVID19/recovered_all_networks_diagnostic_s01e03_de_identified.csv"
+      ]
     }
   ],
   "PhoneNumberUuidTable": {
     "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
     "TableName": "IMAQAL"
   },
+  "TimestampRemappings": [
+    {
+      "TimeKey": "received_on",
+      "ShowPipelineKeyToRemapTo":  "diagnostic_s01e03_raw",
+      "RangeStartInclusive": "2020-08-26T17:00:00+03:00",
+      "RangeEndExclusive": "2020-08-27T13:00:00+03:00"
```

Comment:
The CSV goes to 15:17, are you restricting to 13:00 because we already have those messages anyway?

Reply:
Yes the network resumed at 12 but to be safe have updated to 15:17 thanks.